projects/1.20-fabric/assets/modrinth-big-globe/bigglobe/lang/zh_cn.json

```
@@ -0,0 +1,309 @@
+	"block.bigglobe.voidmetal_block": "虚空金属块",
+	"block.bigglobe.wart_weed": "绯红苗",
+
+	"block.bigglobe.molten_rock_1": "温暖岩石",
+	"block.bigglobe.molten_rock_2": "温暖岩石",
+	"block.bigglobe.molten_rock_3": "炽热岩石",
+	"block.bigglobe.molten_rock_4": "炽热岩石",
+	"block.bigglobe.molten_rock_5": "熔融岩石",
+	"block.bigglobe.molten_rock_6": "熔融岩石",
+	"block.bigglobe.molten_rock_7": "熔融岩石",
+	"block.bigglobe.molten_rock_8": "熔融岩石",
+
+	"block.bigglobe.cloud": "云",
+	"block.bigglobe.red_aura_infused_cloud": "红色息云",
+	"block.bigglobe.orange_aura_infused_cloud": "橙色息云",
+	"block.bigglobe.yellow_aura_infused_cloud": "黄色息云",
+	"block.bigglobe.yellow_green_aura_infused_cloud": "黄绿色息云",
+	"block.bigglobe.green_aura_infused_cloud": "绿色息云",
+	"block.bigglobe.cyan_green_aura_infused_cloud": "青绿色息云",
+	"block.bigglobe.cyan_aura_infused_cloud": "青色息云",
+	"block.bigglobe.cyan_blue_aura_infused_cloud": "青蓝色息云",
+	"block.bigglobe.blue_aura_infused_cloud": "蓝色息云",
+	"block.bigglobe.purple_aura_infused_cloud": "紫色息云",
+	"block.bigglobe.magenta_aura_infused_cloud": "品红色息云",
+	"block.bigglobe.magenta_red_aura_infused_cloud": "紫红色息云",
+	"block.bigglobe.omni_aura_infused_cloud": "全能息云",
+
+	"block.bigglobe.void_cloud": "虚空云",
+	"block.bigglobe.red_aura_infused_void_cloud": "红色虚空息云",
+	"block.bigglobe.orange_aura_infused_void_cloud": "橙色虚空息云",
+	"block.bigglobe.yellow_aura_infused_void_cloud": "黄色虚空息云",
+	"block.bigglobe.yellow_green_aura_infused_void_cloud": "黄绿色虚空息云",
+	"block.bigglobe.green_aura_infused_void_cloud": "绿色虚空息云",
+	"block.bigglobe.cyan_green_aura_infused_void_cloud": "青绿色虚空息云",
+	"block.bigglobe.cyan_aura_infused_void_cloud": "青色虚空息云",
+	"block.bigglobe.cyan_blue_aura_infused_void_cloud": "青蓝色虚空息云",
+	"block.bigglobe.blue_aura_infused_void_cloud": "蓝色虚空息云",
+	"block.bigglobe.purple_aura_infused_void_cloud": "紫色虚空息云",
+	"block.bigglobe.magenta_aura_infused_void_cloud": "品红色虚空息云",
+	"block.bigglobe.magenta_red_aura_infused_void_cloud": "紫红色虚空息云",
+	"block.bigglobe.omni_aura_infused_void_cloud": "全能虚空息云",
+
+	"item.bigglobe.ash": "灰烬",
+	"item.bigglobe.ball_of_string": "线团",
+	"item.bigglobe.chorus_spore": "紫颂孢子",
+	"item.bigglobe.percussive_hammer": "冲击锤",
+	"item.bigglobe.slingshot": "弹弓",
+	"item.bigglobe.soul_lava_bucket": "灵魂熔岩桶",
+	"item.bigglobe.spelunking_rope": "探窟绳",
+	"item.bigglobe.spelunking_rope.tooltip": "系在悬崖边的绳锚上。",
+	"item.bigglobe.sulfur": "硫磺",
+	"item.bigglobe.torch_arrow": "火把箭",
+	"item.bigglobe.public_waypoint": "公共路点",
+	"item.bigglobe.private_waypoint": "私人路点",
+	"item.bigglobe.voidmetal_ingot": "虚空金属锭",
+	"item.bigglobe.voidmetal_upgrade": "锻造模板",
+	"item.bigglobe.voidmetal_helmet": "虚空金属头盔",
+	"item.bigglobe.voidmetal_chestplate": "虚空金属胸甲",
+	"item.bigglobe.voidmetal_leggings": "虚空金属护腿",
+	"item.bigglobe.voidmetal_boots": "虚空金属靴子",
+	"item.bigglobe.voidmetal_upgrade.additions_slot_description": "放置虚空金属锭",
+	"item.bigglobe.voidmetal_upgrade.applies_to": "钻石装备",
+	"item.bigglobe.voidmetal_upgrade.base_slot_description": "放置钻石装备",
```

```suggestion
	"item.bigglobe.voidmetal_upgrade.base_slot_description": "放入钻石盔甲",
```
`放入`为原版用词，`盔甲`是为遵从原文